tests/Benchmarks/Buffers/Writer.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Xunit.Performance;
```

Comment:
It would be good if we have a naming convention for the sub-folders containing tests. In thins case the sub-folder is called  "Buffers", but we have two projects System.Buffers.Primitives and System.Buffers.Experimental". "Buffers" makes it not clear which API project these tests are for. Maybe the folder should be called "SystemBuffersPrimitives" or "BuffersPrimitives" or something like that.